tasks/__main__zipapp.py

```diff
@@ -0,0 +1,159 @@
+import json
+import os
+import sys
+import zipfile
+
+ABS_HERE = os.path.abspath(os.path.dirname(__file__))
+NEW_IMPORT_SYSTEM = sys.version_info[0:2] > (3, 4)
+
+
+class VersionPlatformSelect(object):
+    def __init__(self):
+        self.archive = ABS_HERE
+        self._zip_file = zipfile.ZipFile(ABS_HERE, "r")
+        self.modules = self._load("modules.json")
+        self.distributions = self._load("distributions.json")
+        self.__cache = {}
+
+    def _load(self, of_file):
+        version = ".".join(str(i) for i in sys.version_info[0:2])
+        all_platforms = json.loads(self.get_data(of_file).decode("utf-8")).get(version, {})
+        content = all_platforms.get("any", {})
+        content.update(all_platforms.get(sys.platform, {}))
+        return content
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if False:
+            self._zip_file.close()
```

Comment:
seems left over?

Reply:
yes 👍 will remove, but eases debugging 👍 until I iron out the small remaining bugs